snc.sh

```diff
@@ -321,6 +327,8 @@ sudo systemctl reload NetworkManager
 
 # Create the INSTALL_DIR for the installer and copy the install-config
 rm -fr ${INSTALL_DIR} && mkdir ${INSTALL_DIR} && cp install-config.yaml ${INSTALL_DIR}
+${YQ} write --inplace ${INSTALL_DIR}/install-config.yaml compute[0].architecture ${ARCH}
+${YQ} write --inplace ${INSTALL_DIR}/install-config.yaml controlPlane.architecture ${ARCH}
```

Comment:
looks like for `x86_64` valid value is `amd64` otherwise it fails our current tests fails because of this.

```
"failed to fetch Master Machines: failed to load asset \"Install Config\": invalid \"install-config.yaml\" file: [controlPlane.architecture: Unsupported value: \"x86_64\": supported values: \"amd64\", \"s390x\", \"ppc64le\", compute[0].architecture: Unsupported value: \"x86_64\": supported values: \"amd64\", \"s390x\", \"ppc64le\"
```

Reply:
indeed, this can also be seen in the install-config.yaml changes in this PR

Reply:
fixed with $yq_ARCH variable in squashed commit